django_graph_api/graphql/types.py

```diff
@@ -39,27 +39,45 @@ def __init__(self):
         Field.creation_counter += 1
 
     def get_value(self):
-        if hasattr(self.obj, 'get_{}'.format(self.name)):
-            return getattr(self.obj, 'get_{}'.format(self.name))()
+        raw_value = self.get_raw_value()
+        type = self.type_
```

Comment:
I might prefer to use self.type_ throughout just to avoid overriding the `type` builtin fwiw but it also doesn't really matter for such a small method.